__tests__/workers/cdc.ts

```diff
@@ -452,6 +452,49 @@ describe('user', () => {
     const [state] = await con.getRepository(UserState).find();
     expect(state?.key).toEqual(UserStateKey.CommunityLinkAccess);
   });
+
+  it('should throw an error when creating user state and not related to duplicate entry', async () => {
+    const after: ChangeObject<ObjectType> = {
+      ...base,
+      reputation: submissionAccessThreshold,
+    };
+    after.id = '1234567890123456789012345678901234567'; // 37 characters - exceeds limit
+    try {
+      await expectSuccessfulBackground(
+        worker,
+        mockChangeMessage<ObjectType>({
+          after,
+          before: base,
+          op: 'u',
+          table: 'user',
+        }),
+      );
```

Comment:
You should add a step here that makes sure you don't reach this step. Otherwise, the background worker may be completed successfully, and you wouldn't even know